src/ip_gaussian_grid_mod.f90

```diff
@@ -1,6 +1,12 @@
 !> @file
-!! @brief Gaussian grid
-
+!! Gaussian grid coordinate transformations
```

Comment:
We need to use the @brief tag for files, but not for subroutines/functions.

Reply:
Is it just to be explicit because I don't see a difference in the output.